R-package/tests/testthat/test_lgb.Booster.R

```diff
@@ -385,3 +385,65 @@ test_that("Booster$update() throws an informative error if you provide a non-Dat
         )
     }, regexp = "lgb.Booster.update: Only can use lgb.Dataset", fixed = TRUE)
 })
+
+context("save_model")
+
+test_that("Saving a model with different feature importance types works", {
+    set.seed(708L)
+    data(agaricus.train, package = "lightgbm")
+    data(agaricus.test, package = "lightgbm")
```

Comment:
Sorry for commenting in the closed PR, but is seems that test data is not used in this test.

Reply:
blegh yeah I copied and pasted from other tests. Cut out the `valids` since it wasn't necessary, but I forgot to remove the test data. I'll just bundle that into #3240